apps/velo-external-db/test/e2e/app_data.e2e.spec.ts

```diff
@@ -115,7 +115,7 @@ describe(`Velo External DB Data REST API: ${currentDbImplementationName()}`,  ()
         )
     })
 
-    test('insert api should succeed if item already exists and overwriteExisting is on', async() => {
+    test.skip('insert api should succeed if item already exists and overwriteExisting is on', async() => {
```

Comment:
?

Reply:
These tests will be skipped, It will be changed after #421 will be merged to #352, #421 changed these tests and added new supported tests.

Reply:
It's better to put there if only for Google sheets, instead disable it for all the implementations